lib/presentation/redux/reducers/account_reducer.dart

```diff
@@ -38,6 +39,12 @@ final accountReducer = combineReducers<AccountState>([
       (AccountState state, _) => state.startLoadingState()),
   TypedReducer<AccountState, AccountAction>(
       (AccountState state, AccountAction action) => state.sendViewState(viewState: action.viewState)),
+  TypedReducer<AccountState, GetAccountInformationAction>((AccountState state, GetAccountInformationAction action) =>
+    state.setAccountInformation(
+      newViewState: action.viewState,
+      newUser: action.viewState.fold(
```

Comment:
one remark, if you don't care to failure, why you dont' use `action.viewState.map {}`.

Reply:
oke i fixed it